Resources/Storage/test/Test_StorageElement.py

```diff
@@ -81,6 +81,79 @@ def mock_StorageFactory_generateStorageObject( storageName, pluginName, paramete
 
   return S_OK( storageObj )
 
+def mock_StorageFactory_getConfigStorageName( storageName, referenceType, rootConfigPath ):
+  return S_OK( storageName )
+
+def mock_StorageFactory_getConfigStorageOptions( storageName, rootConfigPath, derivedStorageName = None ):
+  """ Get the options associated to the StorageElement as defined in the CS
+  """
+
+  options = {'BackendType': 'local',
+                            'ReadAccess': 'Active',
+                            'WriteAccess': 'Active',
```

Comment:
Wrong indentations. Also, do you need this one when there's already a `mock_StorageFactory_getConfigStorageOptions` in Test_FilePlugin.py?